src/codegen/riscv64/register-riscv64.h

```diff
@@ -254,7 +260,7 @@ class FPURegister : public RegisterBase<FPURegister, kDoubleAfterLast> {
   explicit constexpr FPURegister(int code) : RegisterBase(code) {}
 };
 
-enum MSARegisterCode {
+enum VRegisterCode {
 #define REGISTER_CODE(R) kMsaCode_##R,
```

Comment:
Perhaps "kVRegCode_##R" is better here.
Also we might change kMsaAfterLast to a proper name.